lib/module/esp-wroom-32.yaml

```diff
@@ -0,0 +1,45 @@
+datasheet: https://espressif.com/en/products/hardware/esp-wroom-32/overview
```

Comment:
I'd rather link directly to the datasheet here https://espressif.com/sites/default/files/documentation/esp_wroom_32_datasheet_en.pdf